lib/chef/provider/powershell_script.rb

```diff
@@ -29,20 +29,6 @@ def initialize (new_resource, run_context)
         add_exit_status_wrapper
       end
 
-      def action_run
-        validate_script_syntax!
-        super
-      end
-
-      def command
```

Comment:
@ksubrama That's all well and good, but powershell_script doesn't care about that either.. it delegates that decision to windows_script.  If we fix that logic, it should be there.